### PR TITLE
ODS-5560 Packages - EdFi.Ods.Populated.Template.SK workflow added

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
EdFi.Ods.Populated.Template.SK Github build Records: 243987 matches with Teamcity buid Records: 243987

File: .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      run: |
+        $ErrorActionPreference = 'Stop'
+        [Environment]::SetEnvironmentVariable('msbuild_buildConfiguration', '${{ env.CONFIGURATION }}')
+        Import-Module -Force -Scope Global "$env:GITHUB_WORKSPACE/Starter-Kit-SEA-Modernization/sample-database-template/modules/create-sk-template.psm1"
+        Initialize-SKTemplate 
+      shell: powershell
+    - name: pack
+      working-directory: ./Starter-Kit-SEA-Modernization/
+      run: |
+        .\build.githubactions.ps1 pack -Configuration ${{ env.CONFIGURATION }} -InformationalVersion ${{ env.INFORMATIONAL_VERSION }} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -NuspecFilePath "$env:GITHUB_WORKSPACE\Starter-Kit-SEA-Modernization\sample-database-template\EdFi.Ods.Populated.Template.Sk.nuspec" -PackageName  "EdFi.Ods.Populated.Template.SK"
```
Comment (bptillman) [inline .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml]: Please switch from using blackslashes to forward slashes in the path to the nuspec file parameter here.

File: .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  repository_dispatch:
+    types: [triggered-from-ods-repo,triggered-from-implementation-repo ,triggered-from-datastandard-repo]
+env:
+  EDFI_STANDARD_REFERENCE: "v4.0.0"
+  INFORMATIONAL_VERSION: "7.0"
```
Comment (bptillman) [inline .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml]: Do we know why the builds in TeamCity were done with version 1.0.x? Changing this to 7.0.x makes sense but I was curious if there was a reasoning this package ended up with such a different version number scheme.

**Review by bptillman — CHANGES_REQUESTED** [review]
The template built here looks correct so the build itself is good. Just one minor change and one question I had and once those are resolved this should be good to go.

File: .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml (hunk 1 above)
Comment (semalaiappan) [reply .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml]: Fixed it . Thanks

File: .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml (hunk 2 above)
Comment (semalaiappan) [reply .github/workflows/Packages - EdFi.Ods.Populated.Template.SK.yml]: Moved back to 1.1.X after discussing with Vinaya on this

File: .github/workflows/Packages - EdFi.Ods.Extensions.SK.yml [hunk 3]
```diff
@@ -6,8 +6,6 @@
 name: Packages - EdFi.Ods.Extensions.Sk
 
 on:
-  pull_request:
```
Comment (semalaiappan) [inline .github/workflows/Packages - EdFi.Ods.Extensions.SK.yml:9]: I removed here because we are not running on pull request for any other templates also

**Review by bptillman — APPROVED** [review]

**Pull request merged**